charts/trino/values.yaml

```diff
@@ -727,6 +727,54 @@ kafka:
   #    }
   # ```
 
+jmx:
+  # https://trino.io/docs/current/admin/jmx.html
+  enabled: false
+  registryPort: 9080
+  serverPort: 9081
+
+jmxExporter:
```

Comment:
How about making this a `exporter` subsection in `jmx`?

Reply:
OTOH Trino also supports OpenMetrics, so the JMX Exporter might not be always necessary: https://github.com/airlift/airlift/blob/master/openmetrics/src/main/java/io/airlift/openmetrics/MetricsConfig.java#L34

OpenMetrics are not documented yet.

This is a nitpick comment, feel free to ignore.